package.json

```diff
@@ -72,7 +70,8 @@
     "ts-loader": "^9.5.1",
     "ts-node": "^10.9.2",
     "tsconfig-paths": "4.2.0",
-    "typescript": "^5.4.3"
+    "typescript": "^5.4.3",
+    "typescript-eslint": "^7.6.0"
```

Comment:
This is the [new package for compatibility](https://typescript-eslint.io/getting-started#step-2-configuration) with v9.x fo ESLint.